src/ant/src/CMakeLists.txt

```diff
@@ -37,19 +37,36 @@ swig_lib(NAME      ant
          SCRIPTS   AntennaChecker.tcl
 )
 
+add_library(ant_lib
+    AntennaChecker.cc
+)
+
 target_sources(ant
   PRIVATE
     MakeAntennaChecker.cc
-    AntennaChecker.cc
+)
+
+target_include_directories(ant_lib
+  PUBLIC
+    ../include
 )
 
 target_include_directories(ant
```

Comment:
We did this one here, generally we tend to try to do "include what you use" instead of assuming transitively exported dependencies. It's a little hard to keep it consistent across all the targets without some kind of "build cleaner" system. If you feel strongly there shouldn't be duplication in the wrapping targets I'll do another pass, but wanted to note this will probably be hard to keep consistent without some cleaner/checker system.